pkg/actors/actors.go

```diff
@@ -389,8 +394,13 @@ func (a *actorsRuntime) callRemoteActorWithRetry(
 	// TODO: Once resiliency is out of preview, we can have this be the only path.
 	if a.isResiliencyEnabled {
 		if a.resiliency.GetPolicy(req.Actor().ActorType, &resiliency.ActorPolicy{}) == nil {
-			policyRunner := resiliency.NewRunner[*invokev1.InvokeMethodResponse](ctx,
+			// This policy has built-in retries so enable replay in the request
+			req.WithReplay(true)
```

Comment:
Right now, Replay is enabled with `isResiliencyEnabled `. As we are planning to make it default path i.e. with resiliency enabled by default, would that mean that streaming be also enabled by default? Or should we make streaming an option with another feature flag?

Reply:
This change is independent from resiliency and it's not "full" streaming support. I don't think there's a way to make this change behind a feature flag since it changes something that is too core and widespread across Dapr...